### PR TITLE
Provide instructions for engineering promotion and compensation increase requests

### PR DESCRIPTION
**Pull request by jeanduplessis**

### CODE REVIEW

File: handbook/engineering/career-development/talent-review-process.md [hunk 1]
```diff
@@ -52,10 +53,28 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
...
-## Considerations for promotion
+## Promotion requests
+
+Promotion requests are made immediately following the [talent review callibration meeting](#calibration-meeting).
+
+The Engineering Manager will complete a promotion request using the [promotion request template](https://docs.google.com/document/d/1DHyPtIooi0J2426iemzzTqLle9UBasrqLZPGET1O7Sc/edit) and submit it for approval.
+
+Following approval from to the relevant decision-makers (EM, Director, VP), the promotion request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the promotion can be communicated to the teammate.
+
+Decisions regarding promotions are prioritized by the decision-makers, and made quickly. Promotions are communicated back to the ICs in a timely manner.
```
Comment (chrispine) [inline handbook/engineering/career-development/talent-review-process.md:64]: In the section below, you include a template for communicating the compensation increase to the IC. Is there a similar (or the same) template for communicating promotions?

**Review by chrispine — APPROVED** [review]
One small question, but these look like really useful additions, thanks!

File: handbook/engineering/career-development/talent-review-process.md (hunk 1 above)
Comment (jeanduplessis) [reply handbook/engineering/career-development/talent-review-process.md:64]: @chrispine I don't have one actually but good callout, I will create one and add it here.

File: handbook/engineering/career-development/talent-review-process.md [hunk 2]
```diff
@@ -52,10 +53,28 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
...
+
+## Compensation increase requests
+
+An in-band compensation increase (while staying at the same level) can happen at any time, in recognition of a teammate exceeding expectations in their current level. 
+
+The Engineering Manager will complete a compensation increase request using the [compensation increase request template](https://docs.google.com/document/d/1nWFcPjCNALww3kwzl_1l6Gjhrpsl1QGZCHb0hTEkQxk/edit) and submit it for approval.
+
+Following approval from to the relevant decision-makers (EM, Director, VP), the compensation increase request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the promotion can be communicated to the teammate using the [compensation increase confirmation template](https://docs.google.com/document/d/1X76t_ZTbmlqp4i9rJHXesYSECxZn5f_MZcUNtpj5aGU/edit).
```
Comment (jeanduplessis) [inline handbook/engineering/career-development/talent-review-process.md:80]:
```suggestion
Following approval from the relevant decision-makers (EM, Director, VP), the compensation increase request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the compensation increase can be communicated to the teammate using the [compensation increase confirmation template](https://docs.google.com/document/d/1X76t_ZTbmlqp4i9rJHXesYSECxZn5f_MZcUNtpj5aGU/edit).
```

File: handbook/engineering/career-development/talent-review-process.md [hunk 3]
```diff
@@ -52,10 +53,28 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
...
 Our [career development framework](framework.md) defines the levels that exist at Sourcegraph. Compensation for each of these levels is derived from Option Impact, a startup compensation tool that Sourcegraph and many other companies use to determine their compensation bands.
 
-## Considerations for promotion
+## Promotion requests
+
+Promotion requests are made immediately following the [talent review callibration meeting](#calibration-meeting).
+
+The Engineering Manager will complete a promotion request using the [promotion request template](https://docs.google.com/document/d/1DHyPtIooi0J2426iemzzTqLle9UBasrqLZPGET1O7Sc/edit) and submit it for approval.
+
+Following approval from to the relevant decision-makers (EM, Director, VP), the promotion request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the promotion can be communicated to the teammate.
```
Comment (jeanduplessis) [inline handbook/engineering/career-development/talent-review-process.md:62]:
```suggestion
Following approval from the relevant decision-makers (EM, Director, VP), the promotion request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the promotion can be communicated to the teammate using the [promotion confirmation template](https://docs.google.com/document/d/1mkdJI6cSXWIHthSN_KQKBFSfxs9RQUg_4VCvwmk8FJQ/edit).
```

**Comment by jeanduplessis** [comment]
@sourcegraph/engineering-managers For your attention and feedback

File: handbook/engineering/career-development/talent-review-process.md [hunk 4]
```diff
@@ -52,10 +53,28 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
...
 Our [career development framework](framework.md) defines the levels that exist at Sourcegraph. Compensation for each of these levels is derived from Option Impact, a startup compensation tool that Sourcegraph and many other companies use to determine their compensation bands.
 
-## Considerations for promotion
+## Promotion requests
+
+Promotion requests are made immediately following the [talent review callibration meeting](#calibration-meeting).
+
+The Engineering Manager will complete a promotion request using the [promotion request template](https://docs.google.com/document/d/1DHyPtIooi0J2426iemzzTqLle9UBasrqLZPGET1O7Sc/edit) and submit it for approval.
+
+Following approval from the relevant decision-makers (EM, Director, VP), the promotion request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the promotion can be communicated to the teammate using the [promotion confirmation template](https://docs.google.com/document/d/1mkdJI6cSXWIHthSN_KQKBFSfxs9RQUg_4VCvwmk8FJQ/edit).
```
Comment (felixfbecker) [inline handbook/engineering/career-development/talent-review-process.md:62]: What role does PeopleOps have here? Are they an _approver_ like the Director, or do they just need to be _informed_ to update payroll etc?

Comment (jeanduplessis) [reply handbook/engineering/career-development/talent-review-process.md:62]: @felixfbecker the latter - it's to ensure all necessary processes relating to the change get made in all the systems.

**Pull request merged**